ui/src/app/components/apps-panel.spec.tsx

```diff
@@ -117,7 +117,7 @@ describe('AppsPanel', () => {
     }
   );
 
-  test.each([RuntimeStatus.Deleted, RuntimeStatus.Error, null])(
+  test.each([RuntimeStatus.Deleted, null])(
```

Comment:
As we discussed, it seems like the previous behavior was incorrect.

Could you add Error to the test above this one, to show that it *does* appear?